src/Microsoft.DotNet.Build.Tasks/PackageFiles/BuildVersion.targets

```diff
@@ -0,0 +1,15 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Where do you expect this file to be imported? Is there any reason to not do this work directly in version.targets?